src/main/resources/seleniumGrid.properties

```diff
@@ -0,0 +1,4 @@
+gridHubURL=http://yourGridURL:gridPort/wd/hub
+#example: gridHubURL=http://192.168.10.10:4444/wd/hub
+browserval=firefox
```

Comment:
No not really, I just could not find the entry point where I could provide the grid instance, perhaps you could please link me to it?